server/text_generation_server/models/causal_lm.py

```diff
@@ -152,13 +205,17 @@ def forward(*args, **kwargs):
     return module
 
 
-def pad_tensors(tensors, paddings, dim, value):
-    for i, (tensor, padding) in enumerate(zip(tensors, paddings)):
-        if padding > 0:
-            pad_shape = (0, 0, 0, padding) if dim == -2 else (0, padding)
-            tensors[i] = torch.nn.functional.pad(tensor, pad_shape, value=value)
-            htorch.core.mark_step()
-    return tensors
+def hpu_graph_fn(fn):
+    class FnModule(torch.nn.Module):
+        def forward(self, *args, **kwargs):
+            return fn(*args, **kwargs)
+    return wrap_in_hpu_graph(FnModule(), disable_tensor_cache=False)
+
+
+def enable_hpu_graphs():
```

Comment:
This function name is a little bit confusing -- probably 'enable_hpu_graphs_on_grouped_move' would be better. BTW can we remove this from that PR as it is not functional?

Reply:
Ok, let's remove it for now and I'll figure out a better name once I figure out how to enable it :)